source/mir/random/flex/internal/area.d

```diff
@@ -169,7 +172,8 @@ body
     {
         if (fabs(z) < constants!S.smallExp)
         {
-            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z / 2 + (z * z) / 6 + (z * z * z) / 24);
+            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z * S(0.5) + (z * z) * one_div_6
```

Comment:
> Can you please explain for me the exp(sh.a) part of this formula?

[See 2.2](http://files.wilzbach.me/mir/reports/tinflex.pdf). Please note that I get different results as in the paper for the approximation with the "Z-trick". Hence the current `area` code is mostly based on their approximations, but I hope that we can use the results from the small report instead (if we all agree they are correct).

>  exp(sh.a) (== exp(alpha) in the paper's notation, right?).

Yes.

> Also, there's some duplication here: you can probably use ivLength instead of an explicit iv.rx - iv.lx, no ... ?

Yes, thanks.